bootstrap/config/config.go

```diff
@@ -487,7 +479,7 @@ func CreateProviderClient(
 
 	// The passed in configStem already contains the trailing '/' in most cases so must verify and add if missing.
 	if configStem[len(configStem)-1] != '/' {
```

Comment:
should this also change to use that constant `pathSep`?

Reply:
It is a character , not a string so didn't change it.

Reply:
ok